scene/gui/line_edit.cpp

```diff
@@ -1622,12 +1622,12 @@ void LineEdit::set_caret_column(int p_column) {
 	}
 
 	// Note: Use two coordinates to fit IME input range.
-	Vector2i primary_catret_offset = get_caret_pixel_pos();
+	Vector2 primary_catret_offset = get_caret_pixel_pos();
```

Comment:
Noticed a pre-existing typo:
```suggestion
	Vector2 primary_caret_offset = get_caret_pixel_pos();
```